src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -45,8 +60,49 @@
                :weight :medium
                :style  {:color (colors/theme-colors colors/neutral-50 colors/neutral-40)}} title]])
 
+(def added (reagent/atom ()))
+
+(defn contact-requests-sheet [group]
+  [:f>
+   (fn []
+     (let [{window-height :height} (rn/use-window-dimensions)
+           safe-area (safe-area/use-safe-area)]
+       [rn/view {:style {:height (- window-height (:top safe-area))}}
+        [rn/touchable-opacity
+         {:on-press #(rf/dispatch [:bottom-sheet/hide])
+          :style
+          {:background-color (colors/theme-colors colors/neutral-10 colors/neutral-80)
+           :margin-left      20
+           :width            32
+           :height           32
+           :border-radius    10
+           :justify-content  :center
+           :align-items      :center
+           :margin-bottom    24}}
+         [quo2/icon :i/close {:color (colors/theme-colors colors/neutral-100 colors/white)}]]
+        [quo2/text {:size   :heading-1
+                    :weight :semi-bold
+                    :style  {:margin-left 20}}
+         (i18n/label :t/add-members)]
+        [contact-list/contact-list {:icon  :check
+                                    :group group
+                                    :added added
+                                    :search? true}]
+        [rn/view {:position           :absolute
+                  :padding-horizontal 20
+                  :padding-vertical   12
+                  :padding-bottom     (+ (:bottom safe-area) 33)
+                  :width              "100%"
+                  :background-color   colors/white
+                  :flex-direction     :row
+                  :bottom             0}
+         [quo2/button {:style    {:flex 1}
+                       :on-press #(rf/dispatch [:bottom-sheet/hide])
+                       :disabled (= (count @added) 0)}
+          (i18n/label :t/save)]]]))])
+
 (defn group-details []
-  (let [{:keys [admins chat-id chat-name color public?]} (rf/sub [:chats/current-chat])
+  (let [{:keys [admins chat-id chat-name color public? muted] :as group} (rf/sub [:chats/current-chat])
         members           (rf/sub [:contacts/current-chat-contacts])
         sectioned-members (prepare-members members)
```

Comment:
as "we shouldn't have any logic or computation in views, should moved to subscription", maybe this one need refactor too? it's not relate to your PR changes, but would u like do something? 😉

Reply:
@qfrank not sure what do you mean? This is only a subscription

Reply:
i mean `prepare-members` just like `prepare-contacts` @OmarBasem